File: P1 2016.2/Questao(1)/python/Q1_2016_2.py
import services.Utils as utilBool


def main():
    array_boolean = [False, True, True, False, False, False, True, True, False]

    array_int = utilBool.Utils.boolean_to_int(array_boolean)
    for x in array_int:
        print(x)


if __name__ == "__main__":
    main()

File: P1 2016.2/Questao(1)/java/Q1_2016_2.java
import services.Utils;

public class Q1_2016_2 {
    public static void main(String[] args) {
        boolean[] arrayBoolean = new boolean[] {
                false,
                true,
                true,
                false,
                false,
                false,
                true,
                true,
                false
        };

        int[] arrayInt = Utils.booleanToInt(arrayBoolean);

        for (int i : arrayInt) {
            System.out.println(i);
        }
    }
}
